parser/tests/test_crawler_current_events.py

```diff
@@ -0,0 +1,54 @@
+# pylint: disable=redefined-outer-name
```

Comment:
do you need this?

Reply:
Yes. Fixtures redefine a variable name that has already been defined in the outer scope